lib/cocoapods-core/specification/linter.rb

```diff
@@ -210,10 +210,6 @@ def _validate_authors(a)
               'from default')
           end
         end
-
-        if a.empty?
```

Comment:
238a282: The `author` attribute is specified in the DSL as `:required => true`, so it's linted automatically and we can rely on that. :)